src/components/LoginForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const LoginForm = () => {
+  const [email, setemail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await axios.post('http://localhost:8081/user/login', { email, password });
```

Comment:
Instead using this axios again here. Can you use our HttpClient.js, it has post. See the Implementation by anshu.

Reply:
![image](https://github.com/learning-project-01/bookstore-view/assets/123824203/3fbb4bbe-fef0-4a58-90a5-0f883d0d03a7)